frontend/src/components/Charts/CandlestickChart.vue

```diff
@@ -46,92 +120,237 @@ export default defineComponent({
     const currentScaleLabel = ref('Today')
     let currentScale        = reactive(getScaleByLabel('Today'))
 
-    const series       = ref(
-      [
+    let currentData: CandleData[] = []
+
+    const option = ref({
+      tooltip: {
+        trigger     : 'axis',
+        axisPointer : {
+          type: 'cross'
+        }
+      },
+      grid: {
+        left   : '10%',
+        right  : '10%',
+        bottom : '15%'
+      },
+      xAxis: {
+        type        : 'category',
+        data        : [],
+        scale       : true,
+        boundaryGap : false,
+        axisLine    : {onZero: false},
+        splitLine   : {show: false},
+        splitNumber : 20,
+        min         : 'dataMin',
+        max         : 'dataMax'
+      },
+      yAxis: {
+        scale     : true,
+        splitArea : {
+          show: true
+        }
+      },
+      dataZoom: [
+        {
+          type  : 'inside',
+          start : 50,
+          end   : 100
+        },
         {
-          name : 'series-1',
-          data : []
+          show  : true,
+          type  : 'slider',
+          top   : '90%',
+          start : 50,
+          end   : 100
         }
+      ],
+      series: [
+        {
+          name      : 'Data',
+          type      : 'candlestick',
+          data      : [],
+          itemStyle : {
+            // color: upColor,
+            // color0: downColor,
+            // borderColor: upBorderColor,
+            // borderColor0: downBorderColor
+          },
+          markPoint: {
+            label: {
+              normal: {
+                formatter (param) {
+                  return param != null ? Math.round(param.value) : ''
+                }
+              }
+            },
+            data: [
+              {
+                name  : 'XX标点',
```

Comment:
😂

Reply:
It will fit nicely with the `petStore` 😅